main_test.go

```diff
@@ -0,0 +1,57 @@
+// +build testrunmain
+
+package main
```

Comment:
Add missing license header @iternity-dotcom

Reply:
Done. I assume I've to put in Minio inc. to the copyright, correct?